src/math/Matrix4.js

```diff
@@ -150,13 +150,17 @@ Object.assign( Matrix4.prototype, {
 
 	}(),
 
-	makeRotationFromEuler: function ( euler ) {
+	makeRotationFromEuler: function ( euler, order ) {
 
 		if ( ! ( euler && euler.isEuler ) ) {
 
 			console.error( 'THREE.Matrix4: .makeRotationFromEuler() now expects a Euler rotation rather than a Vector3 and order.' );
 
 		}
+		if ( order ) {
+
+			throw new Error( 'THREE.Matrix4: .makeRotationFromEuler() no longer accepts order as the second parameter.' );
```

Comment:
Similarly here. I would prefer to add these checks in the tests instead.

Reply:
They are not needed at all. If you prefer, I can remove them completely, test do not test for them. I just added them think if the test code made this mistake, real client code might make it as well.

Reply:
I understand, but yeah, I would prefer not to modify `src` for now.